sdk/test-utils/recorder/package.json

```diff
@@ -67,16 +70,20 @@
     "nock": "^11.7.0",
     "rimraf": "^3.0.0",
     "rollup": "^1.16.3",
-    "@rollup/plugin-commonjs": "^11.0.1",
-    "@rollup/plugin-multi-entry": "^3.0.0",
-    "@rollup/plugin-node-resolve": "^7.0.0",
     "rollup-plugin-shim": "^1.0.0",
     "rollup-plugin-sourcemaps": "^0.4.2",
     "rollup-plugin-terser": "^5.1.1",
     "rollup-plugin-visualizer": "^3.1.1",
     "tslib": "^1.10.0"
   },
   "devDependencies": {
+    "@types/assert": "~1.4.3",
+    "assert": "^1.4.1",
```

Comment:
Personal preference, but I like chai better for assertions - it has better messages, a cleaner surface, and is the same in Node and in the browser.

Reply:
Sounds good! I'll use chai 🖖